integration-cli/docker_cli_run_test.go

```diff
@@ -61,6 +61,14 @@ func (s *DockerSuite) TestRunWithoutMemoryswapLimit(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "--memory-swappiness", "0", "busybox", "true")
```

Comment:
@hqhq 
Okay, To summarize can we have consensus on: 
user specifies --memory-swappiness  <value> => use value
user specifies --memory-swappiness => use deafult = 60
user does not specify --memory-swappiness => inherit the value from parent

so code should be something like below ?
var parsedSwappiness int64
        var flSwappiness int64
        flSwappiness = -1

        if *flSwappinessStr != "" {
                parsedSwappiness, err = strconv.ParseInt(*flSwappinessStr, 10, 64)
                if err != nil || parsedSwappiness < 0 || parsedSwappiness > 100 {
                        return nil, nil, cmd, fmt.Errorf("invalid value:%s. valid memory swappiness range is 0-100", *flSwappinessStr)
                }
                flSwappiness = parsedSwappiness
        } else {
                flSwappiness = 60
        }